internal/internal_versioning_client.go

```diff
@@ -174,6 +216,7 @@ func (o *DescribeTaskQueueEnhancedOptions) validateAndConvertToProto(namespace s
 		TaskQueueTypes:         taskQueueTypes,
 		ReportPollers:          o.ReportPollers,
 		ReportTaskReachability: o.ReportTaskReachability,
+		ReportStats: o.ReportStats,
```

Comment:
Is this available in the current dev server? We won't want to make this available until users can actually use it and we have tested it the same way they might.